geoportailv3/static/js/offline/Configuration.js

```diff
@@ -37,9 +49,16 @@ exports = class extends NgeoConfiguration {
     const sizeOfABgLayer = bgFactor * sizeOfANormalLayer;
     map.getLayers().forEach((layer) => {
       if (!(layer instanceof ol.layer.Vector)) {
-        estimation += this.isBgLayer_(layer, map) ? sizeOfABgLayer : sizeOfANormalLayer;
+        if (this.isBgLayer_(layer, map)) {
+          estimation += (layer.get('label') === 'blank') ? 0 : sizeOfABgLayer;
+        } else {
+          estimation += sizeOfANormalLayer;
+        }
       }
     });
+    if (this.appMymaps_ && this.appMymaps_.getMapId() !== '') {
```

Comment:
With mymaps, content is already downloaded. So it should add nothing, no?

Reply:
Ok.... that's true. In fact, I've understood (~never really read) the warning as "xxMB" will be `downloaded/stored`...
Do you think we need another method to know if there is any layer on the map ?
Or do you want that any weight is added for `mymaps` ?